Detectors/MUON/MID/Conditions/src/DCSNamer.cxx

```diff
@@ -0,0 +1,157 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "MIDConditions/DCSNamer.h"
+#include "MIDBase/DetectorParameters.h"
+#include <array>
+#include <fmt/printf.h>
+#include <iostream>
+#include <set>
+#include <optional>
+#include <stdexcept>
+#include <fmt/format.h>
+
+namespace
+{
+const std::string detPrefix = "MID";
+
+// Run2 (0..71) vs Run3 (11xx, 12xx, 13xx, 14xx) detection element ids
```

Comment:
Since it is commented, can this be removed?

Reply:
I would keep it there as a reminder